### ISSUE TITLE
which loss function results in multi-label classification?

### ISSUE THREAD

**Issue by hminooei** [facebookresearch/fastText#813]
Based on the [documentation](https://fasttext.cc/docs/en/supervised-tutorial.html#multi-label-classification), I'm expecting loss='ova' to result in multi-label classification. But in practice (I'm using python fasttext #version 0.8.22), only loss='ns' results in multi-label classification. Is this a bug in python version or expected?

**Comment by Celebio** [comment]
Hi @hminooei ,
The loss 'ova' results indeed in multi-label classification for training. You must specify k=-1 for prediction  to display as many labels as possible and get their independent probability scores.

I am closing the issue, if it doesn't solve your problem feel free to re-open it.

Best regards,
Onur

**Comment by hminooei** [comment]
Hi @Celebio 

With k=-1 essentially I get the same results as k=2 since I only have two classes. However, the probabilities add up to 1 for all instances I try (I was expecting the probabilities don't add up to 1 at least for some examples) whereas if I set the loss to 'ns' the probabilities barely add to 1 (they are either <1 or >1) which is expected. 

Have you guys tested 'ova' loss in the python package?
